src/main/java/com/exadel/discount/entity/Country.java

```diff
@@ -0,0 +1,26 @@
+package com.exadel.discount.entity;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.*;
+import java.util.List;
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@Entity
+@Table(name = "countries")
+public class Country {
+    @Id
+    @GeneratedValue(generator = "UUID")
+    @GenericGenerator(name = "UUID", strategy = "org.hibernate.id.UUIDGenerator")
+    private UUID id;
+
+    @Column(name = "name", nullable = false)
+    private String name;
+
+    @OneToMany(mappedBy = "country")
```

Comment:
fetch = FetchType.LAZY
will be good here, also

Reply:
Done

Reply:
Ups. It's not visible for me still now. Are U sure?